library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadAdapter.kt

```diff
@@ -20,12 +20,18 @@ import com.chuckerteam.chucker.internal.support.highlightWithDefinedColors
  * We're using a [RecyclerView] to show the content of the body line by line to do not affect
  * performances when loading big payloads.
  */
-internal class TransactionBodyAdapter(
-    private val bodyItems: List<TransactionPayloadItem>
-) : RecyclerView.Adapter<TransactionPayloadViewHolder>() {
+internal class TransactionBodyAdapter : RecyclerView.Adapter<TransactionPayloadViewHolder>() {
+
+    private val items = arrayListOf<TransactionPayloadItem>()
+
+    fun setItems(bodyItems: List<TransactionPayloadItem>) {
```

Comment:
```suggestion
    fun setItems(items: List<TransactionPayloadItem>) {
```

Reply:
That's actually a nit, but still will keep the code consistent.

Reply:
I made it like that to avoid calling `this.items`. Just for readability.